lib/solidus_importer/base_importer.rb

```diff
@@ -19,6 +28,10 @@ def before_import(initial_context)
       initial_context
     end
 
+    ##
+    # Defines a method called when a process row fails
+    def process_row_failure(_processor, context:, error:); end
```

Comment:
This method seems unrelated to the PR/commit, It's probably worth moving it to a different commit, briefly explaining why it's there.